frontend/e2e/pages/patients.js

```diff
@@ -37,6 +32,11 @@ function addPatient(dobFormat) {
   this.section.editPatient.click("@resident");
   this.section.editPatient.click("@healthcareWorker");
   this.section.editPatient.click("@saveButton");
+  this.expect.section("@editPatient").to.contain.text("Last name is required");
+  this.expect.section("@editPatient").to.contain.text("Facility is required");
+  this.section.editPatient.setValue("@lastName", lastName);
+  this.section.editPatient.setValue("@facility", "All facilities");
+  this.section.editPatient.click("@saveButton");
```

Comment:
I sunk a number of hours in trying to debug why the e2e tests failed as is with no luck. It seems that sometihng in the new radio button code is causeing selenium to click the error toast instead of the Do you work in healthcare radio select. I have no idea why this is only showing up now. As far as I can tell the generated DOM is the same